lib/player.js

```diff
@@ -2860,7 +2860,15 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       fullyLoaded.reject(this.videoErrorToShakaError_());
     });
 
-    await fullyLoaded;
+    const timeout = new Promise((resolve, reject) => {
+      const timer = new shaka.util.Timer(reject);
+      timer.tickAfter(/* seconds= */ 30);
```

Comment:
Thanks for fixing this! Is it possible to make this a param so that we can define our own value?

Reply:
Done!